scripts/build.ps1

```diff
@@ -56,6 +56,29 @@ Param(
 
 $ErrorActionPreference = "Stop"
 
+#
+# Git Branch
+#
+$TPB_BRANCH = "LOCALBRANCH"
+$TPB_COMMIT = "LOCALBUILD"
+
+try {
+    $TPB_BRANCH = $env:BUILD_SOURCEBRANCH -replace "^refs/heads/"  
+    if ([string]::IsNullOrWhiteSpace($TPB_BRANCH)) { 
+        $TPB_BRANCH = git -C "." rev-parse --abbrev-ref HEAD
+    }
+}
+catch { }
```

Comment:
Should we log some error message here ?

Reply:
It's not important; an error here should only happen locally if git is not installed already, we can safely ignore the error.